oozie/src/main/java/org/apache/falcon/oozie/OozieBundleBuilder.java

```diff
@@ -106,11 +108,24 @@ public Properties build(Cluster cluster, Path buildPath) throws FalconException
         properties.setProperty(AbstractWorkflowEngine.NAME_NODE, ClusterHelper.getStorageUrl(cluster));
 
         //Add libpath
-        String libPath = getLibPath(buildPath);
-        if (libPath != null) {
-            properties.put(OozieClient.LIBPATH, getStoragePath(libPath));
+        String path = getLibPath(buildPath);
```

Comment:
I meant to say that this new section of code for adding storage LibPath, into Properties, from line 110-125 is inside the function Properties build(). So I am of the opinion that this section of code for storageLibPath should be defined in a String returned function and then add the returned string OozieClient.LIBPATH.